### PR TITLE
fix(TBD-10340): [Spark Streaming, CDH 6.1] NoClassDefFoundError on tHiveOutput component

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [X] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [X] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)
TBD-10340

**What is the new behavior?**
Fixed

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 1]
```diff
@@ -13038,8 +13038,8 @@
       "context" : "plugin:org.talend.hadoop.distribution.cdh5x",
       "excludeDependencies" : "true",
       "id" : "DYNAMIC_Cloudera_CDH6_1_1_hive_exec",
-      "mvn_uri" : "mvn:org.talend.libraries/hive-exec-shade-2.1.1-cdh6.1.1/6.0.0",
-      "name" : "hive-exec-shade-2.1.1-cdh6.1.1.jar",
+      "mvn_uri" : "mvn:org.talend.libraries/hive-exec-shade-2.1.1-cdh6.1.1-talend/6.0.0",
+      "name" : "hive-exec-shade-2.1.1-cdh6.1.1-talend.jar",
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:13042]: Has the template file been changed? There is a rule to update template file and then re-generate built-in file based on template updates. Manual changes aren't allowed in built-in file.

Comment (amedvedenko) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:13042]: https://wiki.talend.com/display/rd/Dynamic+Distributions

Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:13042]: Template fixed

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "distribution" : "CLOUDERA",
     "id" : "Cloudera_CDH6_1_1",
     "name" : "Cloudera CDH6.1.1",
+    "projectTechnicalName" : "local",
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:7]: it should be deleted, wiki https://wiki.talend.com/display/rd/Dynamic+Distributions

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 3]
```diff
@@ -12024,7 +12025,7 @@
       "context" : "plugin:org.talend.hadoop.distribution.cdh5x",
       "excludeDependencies" : "true",
       "id" : "DYNAMIC_Cloudera_CDH6_1_1_avro_mapred_1_8_2_cdh6_1_1_hadoop2_jar",
-      "mvn_uri" : "mvn:org.apache.avro/avro-mapred/1.8.2-cdh6.1.1/jar/hadoop2",
+      "mvn_uri" : "mvn:org.apache.avro/avro-mapred/1.8.2-cdh6.1.1//hadoop2",
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:12028]: it's an issue, could you please manually add /jar ? It's exactly generation issue but it should be fixed so I suggest jsut manually change it/ Maybe @aparent-talend can tell something more.

File: main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json [hunk 4]
```diff
@@ -13030,27 +13007,26 @@
...
       "context" : "plugin:org.talend.hadoop.distribution.cdh5x",
       "excludeDependencies" : "true",
-      "id" : "DYNAMIC_Cloudera_CDH6_1_1_hive_exec_spark",
-      "mvn_uri" : "mvn:org.spark-project.hive/hive-exec/1.2.1.spark2",
-      "name" : "hive-exec-1.2.1.spark2.jar",
-      "tagName" : "libraryNeeded",
-      "useStudioRepository" : "true"
+      "id" : "DYNAMIC_Cloudera_CDH6_1_1_hive_exec_1_2_1_jar",
+      "mvn_uri" : "mvn:org.spark-project.hive/hive-exec/1.2.1",
+      "name" : "hive-exec-1.2.1.jar",
```
Comment (amedvedenko) [inline main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:13028]: strange version change. Could you please notify QA to verify Hive as a possible regression?

Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.cdh5x/resources/builtin/cdhx/Cloudera_CDH6_1_1.json:13028]: Will do

**Pull request merged**